### PR TITLE
moving demo files to namespace

### PR DESCRIPTION
**Pull request by larslunde-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by thomast1906 — APPROVED** [review]

**Comment by thomast1906** [comment]
Actually, just noticed duplicate entry - this isn't needed, already added?

https://github.com/hmcts/cnp-flux-config/blob/c5949f844e31006d5d610c854c6ba96748451718/k8s/demo/common-overlay/fees-pay/kustomization.yaml#L12

**Review by thomast1906 — CHANGES_REQUESTED** [review]
Not needed, already referenced

https://github.com/hmcts/cnp-flux-config/blob/c5949f844e31006d5d610c854c6ba96748451718/k8s/demo/common-overlay/fees-pay/kustomization.yaml#L12

File: k8s/demo/common-overlay/fees-pay/kustomization.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ patchesStrategicMerge:
 - ../../../namespaces/fees-pay/ccpay-bubble-frontend/demo.yaml
 - ../../../namespaces/fees-pay/ccpay-paymentoutcome-web/demo.yaml
 - ../../../namespaces/fees-pay/ccpay-payment-api/demo.yaml
+- ../../../namespaces/fees-pay/ccpay-callback-function/demo.yaml
```
Comment (thomast1906) [inline k8s/demo/common-overlay/fees-pay/kustomization.yaml:23]: Can you remove this as part?

Comment (larslunde-hmcts) [reply k8s/demo/common-overlay/fees-pay/kustomization.yaml]: I have removed it now, had not seen the initial existing entry.

**Review by AnilDhawaleGitHub — APPROVED** [review]

**Pull request merged**